example/HMI/app/views/main.handlebars

```diff
@@ -155,15 +155,22 @@ don't worry about this one could be broken -->
 
 <div>
     Multi Page Select
-    {{#tmplit 'MultiSelect' maxColumns=10 data-var-name="tmplitTest:tmplit.MultiSelect_INT" data-var-name-field="tmplitTest:tmplit.MultiSelect_STRING"}}
+    {{#tmplit 'MultiSelect' maxColumns=11 data-var-name="tmplitTest:tmplit.MultiSelect_INT" data-var-name-field="tmplitTest:tmplit.MultiSelect_STRING"}}
```

Comment:
What does maxColumns mean?

Reply:
In this case it's the number of buttons in a column before wrapping